lib/rules/no-magic-numbers.js

```diff
@@ -42,12 +42,12 @@ module.exports = function(context) {
         enforceConst = !!config.enforceConst;
 
     /**
-     * Returns whether the node is number literal
-     * @param {Node} node - the node literal being evaluated
-     * @returns {boolean} true if the node is a number literal
+     * Returns whether the value is number
+     * @param {*} value - the value being evaluated
+     * @returns {boolean} true if value is a number
      */
-    function isNumber(node) {
-        return typeof node.value === "number";
+    function isNumber(value) {
```

Comment:
I think it's cleaner with the two functions. Note, I didn't ask for the function to be removed, just to leave it alone. :)